SpatialGDK/Source/SpatialGDKServices/Private/SpatialGDKServicesModule.cpp

```diff
@@ -137,16 +139,37 @@ void FSpatialGDKServicesModule::ExecuteAndReadOutput(const FString& Executable,
 	FPlatformProcess::ClosePipe(0, WritePipe);
 }
 
+void FSpatialGDKServicesModule::SetProjectName(const FString& InProjectName)
```

Comment:
I guess the regex should be here and we return a bool

Reply:
Hmm it's a bit tricky because this is in the services module and it doesn't have access to the editor module so I would have to move the check here but then we have the checks split all over the place so I think it's easier to keep the regex check in the editor module and assume it's valid when it gets here.